src/accelerate/commands/utils/cursor.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
```

Comment:
If this is heavily inspired by another lib, it needs to be reflected in the copyright probably.